generation/WinSDK/enums.json

```diff
@@ -34862,6 +34862,52 @@
       }
     ]
   },
+  {
+    "name": "CDN_CONSTANTS",
```

Comment:
Perhaps better named this? (https://learn.microsoft.com/windows/win32/dlgbox/common-dialog-box-notifications)

```suggestion
    "name": "COMMON_DIALOG_NOTIFICATIONS",
```

Reply:
Should be singular not plural if this isn't a flags enum.